src/EditorFeatures/CSharpTest2/Recommendations/VoidKeywordRecommenderTests.cs

```diff
@@ -642,5 +642,51 @@ public class VoidKeywordRecommenderTests : KeywordRecommenderTests
         {
             await VerifyAbsenceAsync(@"class c { async async $$ }");
         }
+
+        [Fact]
+        [WorkItem(8617, "https://github.com/dotnet/roslyn/issues/8617")]
+        [Test.Utilities.CompilerTrait(Test.Utilities.CompilerFeature.LocalFunctions)]
+        public async Task TestLocalFunction()
+        {
+            await VerifyKeywordAsync(@"
+class C
+{
+    void M()
+    {
+        $$
+    }
+}");
+        }
+
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/14525")]
```

Comment:
so much of hte keyword recommending system works totally by looking for token patterns.  The idea being that generally teh parse tree is not in a good state, especially when you have random incomplete snippets of text during typing.  So i'm somewhat surprised we wouldn't be able to handle this case, or that we would be thrown off by the parser doing something like this :)